src/state/index.ts

```diff
@@ -3,17 +3,21 @@ import { createBrowserHistory, History } from 'history';
 import { combineReducers, createStore, Reducer } from 'redux';
 import { composeWithDevTools } from 'redux-devtools-extension';
 import { counterReducer, CounterState } from './counter/counter.reducer';
+import { homepageReducer, HomepageState } from './homepage/homepage.reducer';
+import { preloadedState } from './preloaded.state';
 
 export interface AppState {
     counterState: CounterState;
+    homepageState: HomepageState;
     router: RouterState;
 }
 
 export const reducers = (history: History<any>): { [key in keyof AppState]: Reducer<any, any> } => ({
     counterState: counterReducer,
+    homepageState: homepageReducer,
     router: connectRouter(history),
 });
 
 export const history = createBrowserHistory({ basename: 'localsurvey' });
 
-export const store = createStore(combineReducers(reducers(history)), composeWithDevTools());
+export const store = createStore(combineReducers(reducers(history)), preloadedState, composeWithDevTools());
```

Comment:
Was thinking of conditionally loading based on `.env` file, but not sure how that relates to a demo deploy to GH pages?

Reply:
We can set a variable `ENV` to `development` and only populate the store when `ENV === development`, that way we don't touch anything

Reply:
We want to have a working demo on GH-pages. How would we go about that then?

Reply:
Do we want a demo or an actually working product on GH pages? Maybe a single demo survey that can be removed easily.

Also, ENV isn't limited to just `development` or `production`, we could add a third `demo` option that only populates but is basically a production build.

Reply:
Can we pass a .env to deploy the correct environment to GH-pages?

Reply:
This is fine by me

Reply:
We could suggest to load an example survey when the user first loads the app. Combining "show the real product" and "provide a demo" on GH-pages.